src/vm/dllimport.cpp

```diff
@@ -6563,6 +6563,21 @@ class LoadLibErrorTracker
     DWORD   m_priorityOfLastError;
 };  // class LoadLibErrorTracker
 
+static HMODULE LocalLoadLibraryHelperInternal(LPCWSTR name, DWORD flags)
```

Comment:
If I pass non-0 flag over to Linux PAL version of LoadLibraryEx it's going to assert, and I like having the assert to catch people passing non-0 flag in other cases in Linux. So I'm creating a helper that don't pass the flag in non-windows and abstract the difference only for p/invoke scenarios.